config/install/field.field.node.starships.field_passen.yml

```diff
@@ -0,0 +1,22 @@
+langcode: en
+status: true
+dependencies:
+  config:
+    - field.storage.node.field_passen
+    - node.type.starships
+id: node.starships.field_passen
+field_name: field_passen
```

Comment:
This machine name doesn't match the content model. Change here or in the content model so they match.
![starships_field_passengers_machine_name](https://github.com/notS0Solo/swapimigration/assets/1434979/530b1a74-7f8b-4929-aa4e-310a2666967a)